src/config.ts

```diff
@@ -15,7 +15,7 @@ export let config: IConfig = {
   sensitiveAttributes: [],
   timeToYield: 50,
   instrument: false,
-  cssRules: false,
+  skipCookie: false,
```

Comment:
skipCookie -> disableCookie

Reply:
Looks like Slava beat me to this change. He called it allowIdCookie; if we need to change it we can. 

---
In reply to: [243138961](https://github.com/Microsoft/clarity-js/pull/116#discussion_r243138961) [](ancestors = 243138961)